exp/sampler.jl

```diff
@@ -30,6 +31,7 @@ const NAME2OPTIMIZER = Dict(
     "Ipopt" => Ipopt.Optimizer,
     "Mosek" => Mosek.Optimizer,
     "HiGHS" => HiGHS.Optimizer,
+    "SCS" => SCS.Optimizer,
```

Comment:
Instead of SCS, use Clarabel with `static_regularization_constant` set to `1e-7`.
This does the job for me on the 5, 14 and 30-bus system.
Another possibility is to use Clarabel128